### PR TITLE
feat: support /* rtl:ignore */

### LINKED ISSUE #5: rtlcss-directives
Is there a roadmap/timeplan for the rtlcss-directives?

### PR DESCRIPTION
**Pull request by joscha**
This PR adds support for `/* rtl:ignore */`

closes #5 

cc @chrisdoble @hearnden @marcfallows

### CODE REVIEW

File: src/decls.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ const rtlifyDecl = ( decl, keyframes ) => {
     if ( decl.prop.match( /animation/ ) ) {
         value = getProcessedKeyframeValue( decl, keyframes, 'rtl' )
     } else {
-        const important = decl.important;
-        decl.important = false;
+        const important = decl.important
```
Comment (joscha) [inline src/decls.js]: these were fixed by running `yarn lint -- --fix`.

**Comment by vkalinichev** [comment]
Great thanks!

**Comment by joscha** [comment]
awesome @vkalinichev, can you ping here when that feature has made it into a release, please?

**Comment by vkalinichev** [comment]
It's in v1.0.0 already

**Comment by joscha** [comment]
\o/ @vkalinichev - unfortunately I found a small bug - fix is in #17

**Pull request merged**